interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/conference-room.twig

```diff
@@ -0,0 +1,64 @@
+<div class="row conference-room bg-dark">
+        <div class="col-12 d-flex">
+            <div id="local-video-container" class="col-lg-2 col-md-2 col-sm-3 col-4 p-0">
+                <video class="bg-primary" id="local-video"></video>
+            </div>
+            <div class="remote-video-container text-center w-100">
+                <video id="remote-video" class="remote-video d-none mw-100"></video>
+                <div class="waiting-container">
+                    <h1 class="h1-responsive text-light text-center">{{ "Waiting for "|xlt }}<em>{{ calleeName }}</em> {{ "to join"|xlt }} <div class="spinner-border" role="status">
```

Comment:
no spaces at end of translation contants. also splitting strings make it pretty much impossible to internationalize even though it may be a bit awkard in english. maybe something like:
waiting for following to join: calleename

Reply:
Reworded this and fixed the issue.